flow/util/markDontUse.py

```diff
@@ -22,10 +22,10 @@
 
 # Read input file
 print("Opening file for replace:",args.inputFile)
-if args.inputFile.endswith(".gz"):
-    f = gzip.open(args.inputFile, 'rt')
+if args.inputFile.endswith(".gz") or args.inputFile.endswith(".GZ"):
+    f = gzip.open(args.inputFile, 'rt', encoding="ascii")
 else:
-    f = open(args.inputFile)
+    f = open(args.inputFile, encoding="ascii")
```

Comment:
Based on your comments today, do you need to use utf8?

Reply:
When I read it as acsii, the characters are ignored, so it has the same effect as reading it as uft-8 and then converting (except being a little simpler)

Reply:
I was wrong, I had the wrong version of the file copied in